dizoo/common/policy/md_ppo.py

```diff
@@ -1,57 +1,113 @@
-from typing import List, Dict, Any, Tuple, Union, Optional
-import torch
-import torch.nn as nn
-import torch.nn.functional as F
-from ding.rl_utils import ppo_data, ppo_error
-from ding.policy import PPOPolicy
+from typing import List, Dict, Any, Tuple, Union
+from collections import namedtuple
+
+from ding.torch_utils import Adam, to_device
+from ding.rl_utils import ppo_data, ppo_error, ppo_policy_error, ppo_policy_data, v_nstep_td_data, v_nstep_td_error
 from ding.utils import POLICY_REGISTRY
-from ding.model.actor_critic.value_ac import ValueAC
+from ding.policy import PPOOffPolicy
+from ding.policy.common_utils import default_preprocess_learn
+
 
+@POLICY_REGISTRY.register('md_ppo_offpolicy')
+class MultiDiscretePPOOffPolicy(PPOOffPolicy):
 
-class SepValueAC(ValueAC):
+    def _forward_learn(self, data: dict) -> Dict[str, Any]:
```

Comment:
i think  it would be better to add complete comments of overview, arguments and returns for each class functions like [this](https://github.com/opendilab/DI-engine/blob/22a13ab513f78e78ae7b0e08f2ed282e29b6c726/ding/policy/dqn.py#L133).

Reply:
ppo deleted